database/database_test.go

```diff
@@ -30,7 +29,7 @@ func ExampleTempLocalDatabase() {
 func TestMain(m *testing.M) {
 	localDatabase, shutdown, err := TempLocalDatabase()
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "could not initialize local backend %s", err)
+		fmt.Fprintf(os.Stderr, "could not initialize local backend because %s", err)
```

Comment:
Very minor styling nit: I think the most common way to print this out is just add a : between your explanation and the error (so "could not initialize local backend: %s" in this case)

Reply:
Done